patches/metalsmith-in-place+1.4.4.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/node_modules/metalsmith-in-place/lib/index.js b/node_modules/metalsmith-in-place/lib/index.js
```

Comment:
you can test that this diff is correct by checking https://codeload.github.com/superwolff/metalsmith-in-place/tar.gz/7cb06e54142b8843f35178ceb5560946ae356049 which still exists today

I looked into updating to a newer version of metalsmith-in-place, but they added a bunch of dependencies that seemed hard to migrate towards